templates/artist/artist_list.html

```diff
@@ -10,6 +10,7 @@
     <div class="break">
         <hr>
     </div>
+    <div id="filter-options">
```

Comment:
If you opened a new `<div>` without closing it, that means the whole rest of the file has these styles applied. Is that what you want? (Probably not.)